src/Cleaner/BookCleanerEpub.php

```diff
@@ -255,6 +255,8 @@ protected function setLinks( DOMDocument $dom ) {
 				$node->setAttribute( 'href', 'http:' . $href );
 			} elseif ( substr( $href, 0, 1 ) === '/' ) {
 				$node->setAttribute( 'href', $this->baseUrl . $href );
+			} elseif ( substr( $href, 0, 2 ) === './' ) {
+				$node->setAttribute( 'href', $this->baseUrl . '/wiki/' . substr( $href, 2 ) );
```

Comment:
The same link changing needs to be done in `Refresh::getAboutXhtmlWikisource()`, although perhaps this link-fixing should all be done in PageParser.